lib/features/settings/QA.md

```
@@ -192,3 +193,36 @@ $ curl -s https://bannzai.github.io/kashakeibo/PrivacyPolicy-en | grep -o '<h1[^
+
+---
+
+## 3. アカウント
+
+アカウント連携の QA 項目は PR #68 (ブランチ qa-run-20260822) で「## 2. アカウント (バックアップ・プラン・削除)」として整備中。ここでは issue #66 で修正した Google リンクの項目だけを先に記録する (マージ時は PR #68 側のアカウント節へ統合する)。
+
+- [x] **Google サインイン UI の表示**: 設定画面の「Googleでリンク」をタップすると Google のサインイン UI が表示される (実アカウントの入力・リンク完了までは行わない)
+  - 自動化: manual (シート表示までを agent のシミュレータ操作で確認する。リンク完了は実アカウントが必要なため対象外)
+  - 機械検査: `ios/RunnerTests/GoogleSignInURLSchemeTests.swift` がビルド済み Runner.app の Info.plist に REVERSED_CLIENT_ID の URL スキーム・GIDClientID・共有 Extension のスキームが揃っているかを検証する
+
+#### 動作確認
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)